test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -8,24 +8,22 @@ import {
   getChainBalance,
   fundLnd,
   checkIsBalanced,
-  getUserWallet,
+  getAndCreateUserWallet,
   mineAndConfirm,
   sendToAddressAndConfirm,
   waitUntilBlockHeight,
 } from "test/helpers"
 import { getWalletFromRole } from "@core/wallet-factory"
 import * as Wallets from "@app/wallets"
 
-jest.mock("@services/phone-provider", () => require("test/mocks/phone-provider"))
-
 let bitcoindOutside
 
 beforeAll(async () => {
   // load funder wallet before use it
-  await getUserWallet(4)
+  await getAndCreateUserWallet(4)
 
   // "bankowner" user
-  await getUserWallet(14)
+  await getAndCreateUserWallet(14)
```

Comment:
now that we have `00-create-mandatory-users.spec.ts ` is this necessary?

Reply:
seems `00-create-mandatory-users.spec.ts` is no longer needed. was probably a transient issue while I was working on this PR. deleting